contributors/design-proposals/job.md

```diff
@@ -106,13 +110,13 @@ type JobSpec struct {
     Completions *int
 
     // Optional duration in seconds relative to the startTime that the job may be active
-    // before the system tries to terminate it; value must be positive integer
+    // before the system tries to terminate it; value must be a positive integer.
     ActiveDeadlineSeconds *int
 
-    // Optional number of retries, before marking this job failed.
+    // Optional number of retries before marking this job failed.
     BackoffLimit *int
 
-    // Optional time (in seconds), how log a job should be retried before marking it failed.
+    // Optional time (in seconds) specifying how long a job should be retried before marking it failed.
```

Comment:
- [ ] Obviously this duration shouldn't be negative (in which case I assume we'd ignore it, or reject it), but what about zero?  
  Since it's a pointer, the pointer being nil means that it's not set, but what if it's present but zero? Is that the same as it not being set, or is that a valid duration meaning that we won't wait any amount of time for a job complete (that it must complete immediately)?